usmqe/web/tendrl/cluster_work.py

```diff
@@ -12,7 +12,7 @@
 from usmqe.web.tendrl.clusters.pages import\
     ClustersList
 from usmqe.web.tendrl.clusters.import_cluster_wizard.pages\
-    import ImportCluster
+    import ImportClusterSummary
```

Comment:
add some note that this is probably wrong, it's just for import to be functional